website/src/categories/components/toggle-group/accessibility.md

```diff
@@ -0,0 +1,56 @@
+|---|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|Tab| 	Shifts focus to the first toggle button in a toggle group. Or when a toggle button is focused the focus will go the the next focusable element in the page when the tab key is pressed.                                                                                                             |
+|Space/Enter | Selects/deselects the currently focused option. In non-multiple variant selecting one toggle button deselects another one, when there is already another one selected.                                                                                                                               |
+|Arrow Keys	| Once you are "in" a toggle group you can navigate to the next toggle button (option) by using the right or down arrow key. You can navigate back to the previous one with left or up. The focus indicator loops, so when you are at the last option and press "down" it will focus the first option. |
+
+{.ds-table .ds-table-align-top}
+
+</section>
+
+<section>
+
+## WAI-ARIA
+
+WAI-ARIA Roles, States, and Properties for a toggle group provide essential information to assistive technologies and screen readers. They convey the toggle group's role and additional properties to ensure accessibility and a better user experience for individuals using assistive technology.
+
+
+|Attribute | Value | Description                                                                                                                                                                                             | User supplied <sl-icon name="info" aria-describedby="tooltip1" size="md"></sl-icon><sl-tooltip id="tooltip1">Specifies whether the attribute is always set in the component (no) or it needs to be provided by the developer (yes)</sl-tooltip>|
+|-|-|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-|
+|`role`	|`region`| Identifies the toggle group as a significant element of the page and that it should be easily accessible with screen readers.                                                                           |no|
+|`aria-label`|string| Provide an accessible name for the toggle button in it when an there is an icon-only variant used. The `aria-label` can be also added for the toggle group to describe the purpose of the toggle group. |yes|
+|`aria-labelledby`|string| Can be used to connect with a single header/element that describes the toggle group and there is no `aria-label`.                                                                                        |yes|
```

Comment:
```suggestion
|`aria-labelledby`|string| Can be used to connect with a single (heading) element that describes the toggle group, instead of using`aria-label`, which would duplicate the text.                                                                                        |yes|
```